package.json

```diff
@@ -40,6 +43,7 @@
     "yargs": "^17.0.1"
   },
   "devDependencies": {
+    "@braintree/sanitize-url": "^7.1.1",
```

Comment:
`@braintree/sanitize-url`, `dayjs` and `debug` were needed to make `withMermaid()` work in vitepress.

Maybe related: emersonbottero/vitepress-plugin-mermaid#83